### PR TITLE
[MIRROR] Flavour Enhancement: Crates

### PR DESCRIPTION
**Pull request by VirgoBot**
Original PR: https://github.com/PolarisSS13/Polaris/pull/7342
--------------------
Added 54 lore-tastic crate variants. Almost every TSC + some new subsidiaries and brandings.
Almost every cargo orderable uses one of the new crates.
Also replaces most "generic" crates in maintenance with a randomised version which picks from a list of basic (non large, non secure) crates excluding medical and luxury types.

This pic includes most of them:
crates
Einstein Large, Xion Large, NT Large, Aether Large, Heph Large (sec), Aether Lsec, NT Lsec, Einstein Lsec, Xion Lsec

NT sec, NanoCare, ZengHu, Veymed, Gilthari, Focal Point, Xion, Einstein

WT, Morpheus, Cyber Solutions, Bishop (logo has been edited since), Grayson, Heph, Lawson Arms, SAARE

NT Hazardous, Centauri Freezer, NT Freezer, Aether, Gilthari, Centauri, Focal Point, Einstein
Xion, Grayson, Morpheus, WT, Heph, Oculum Broadcast, Zenghu, Veymed

NT, NanoThreads, NanoCare, AlliCo, Hedberg-Hammarstrom, Galaksi Appliance, UmMarcar Office Supply, Moghes Imported Sissalik Jerky

Not pictured: Aether secure, SCG, Thinktronic, Consolidated Agriculture, NanoTrasen Pastures large.

ALSO as requested: Crates now drag length-ways and now have rotation verbs to allow easier placement. By default they still 'spawn' facing traditionally 'forward'.

### CODE REVIEW

File: code/datums/supplypacks/engineering.dm [hunk 1]
```diff
@@ -11,30 +11,75 @@
...
 /datum/supply_pack/eng/smescoil/super_io
 	name = "Superconducting Transmission Coil"
 	contains = list(/obj/item/weapon/smes_coil/super_io)
 	cost = 90
-	containertype = /obj/structure/closet/crate/engineering
+	containertype = /obj/structure/closet/crate/focalpoint
 	containername = "Superconducting Transmission Coil crate"
 
+<<<<<<< HEAD
+=======
```
Comment (lbnesquik) [inline code/datums/supplypacks/engineering.dm:39]: Reeee

File: code/datums/supplypacks/engineering.dm [hunk 2]
```diff
@@ -11,30 +11,75 @@
...
+	containername = "shield hull generator crate"
+
+/datum/supply_pack/eng/hull_shield/advanced
+	name = "Advanced Hull Shield Generator"
+	contains = list(/obj/machinery/shield_gen/external/advanced)
+	cost = 120
+	containertype = /obj/structure/closet/crate/focalpoint
+	containername = "advanced hull shield generator crate"
+
+>>>>>>> d7f25c3... Flavour Enhancement: Crates (#7342)
```
Comment (lbnesquik) [inline code/datums/supplypacks/engineering.dm:82]: Reeeeeee

File: code/datums/supplypacks/engineering.dm [hunk 3]
```diff
@@ -127,10 +172,11 @@
 			/obj/structure/particle_accelerator/power_box,
 			/obj/structure/particle_accelerator/end_cap
 			)
-	containertype = /obj/structure/closet/crate/secure/engineering
+	containertype = /obj/structure/closet/crate/secure/einstein
 	containername = "Particle Accelerator crate"
 	access = access_ce
 
+<<<<<<< HEAD
```
Comment (lbnesquik) [inline code/datums/supplypacks/engineering.dm:179]: Reeeeeeeeeee

File: code/datums/supplypacks/engineering.dm [hunk 4]
```diff
@@ -144,6 +190,31 @@
 	cost = 80
 	containertype = /obj/structure/closet/crate/engineering
 	containername = "shield generator construction kit crate"
+=======
```
Comment (lbnesquik) [inline code/datums/supplypacks/engineering.dm:193]: rreeeeeeeeeeee

File: code/datums/supplypacks/engineering.dm [hunk 5]
```diff
@@ -144,6 +190,31 @@
...
+	access = access_ce
+
+/datum/supply_pack/eng/shield_cap
+	contains = list(/obj/item/weapon/circuitboard/shield_cap)
+	name = "Bubble shield capacitor circuitry"
+	cost = 30
+	containertype = /obj/structure/closet/crate/secure/focalpoint
+	containername = "shield capacitor circuitry crate"
+	access = access_ce
+>>>>>>> d7f25c3... Flavour Enhancement: Crates (#7342)
```
Comment (lbnesquik) [inline code/datums/supplypacks/engineering.dm:217]: ree

**Comment by Novacat** [comment]
I am aware, but I have a feeling that a lot of it is due to unmarked Vorestation edits that will need to be sorted out, so a proper fix will require booting up notepad++ and gitkraken and is not as simple as just a few text edits

**Pull request merged**